modules/installation-aws-config-yaml.adoc

```diff
@@ -85,9 +87,23 @@ array. Because the sections have different definitions, the first line of the
 single machine pool, it is possible that future versions of {product-title}
 will support defining multiple compute pools during installation. Only one
 control plane pool is used.
-<4> To configure faster storage for etcd, especially for larger clusters, set the
+<4> Whether to enable or disable simultaneous multithreading, or
+`hyperthreading`. By default, simultaneous multithreading is enabled
+to increase the performance of your machines' cores. You can disable it by
+setting the parameter value to `Disabled`. If you disable simultanous
+multithreading in some cluster machines, you must disable it in all cluster
+machines.
```

Comment:
Do you know the source of this all-or-none condition?  I can't see why it couldn't be disabled on a per-machine-pool basis.

Reply:
Before 4.1 GA, Abhinav said "machine-config-operator doesn't support upgrading such a change {after installation} now" in a DM. If you know that this is wrong because of you can disable it on a per-machine-pool-basis after installation in 4.2 or want to expose creating more machines at installation, please file a bug.